### PR TITLE
Enable multiple tourguides

### PR DESCRIPTION
**Pull request by pointmekin**
# Note (Please read)
Updated
- Due to the limitation of the `rn-tourguide` library, there could only be 1 tourguide inside the application. This means that in the case where settings has 4 steps, and home has 7 steps, when you click through the 4 steps in the settings page, it will continue with the fifth step that belongs to the home screen guide. This is because the step is shared throughout the whole application.
- This doesn't allow us to follow the design. 
- I found. a fork of this repository that enables the project to contain multiple tourguides. This is the link to that repository: https://github.com/b-steel/rn-tourguide/tree/multiple-tours
- Please refer to this github thread: https://github.com/xcarpentier/rn-tourguide/issues/68
- I then create a new folder inside the src folder called "library" this will contain any external libraries that we want to hijack and include it in our project in the future. For compatibility and efficiency reasons, eslint rules will be disabled for this folder. 
- I have copied the content in the src folder of this forked repository and include it in `src/library/rn-multiple-tourguide` folder of our project.

Please check it out, or try running this in your emulator to ensure that everything's working fine. The expected result is that the tourguide on the home page and the settings page should follow their respective designs in figma. And it should not show any steps that doesn't exist in its own page.

This is the result of implementing this change:

https://user-images.githubusercontent.com/53035529/160285167-9a6744cf-ce2e-46b1-8f3e-df2aa0e1701a.mov

### CODE REVIEW

**Comment by chuwong35122** [comment]
Could you post the screen recording for the change? And discuss more on 1 tour guide instance in the application for the team too?

**Comment by pointmekin** [comment]
> Could you post the screen recording for the change? And discuss more on 1 tour guide instance in the application for the team too?

Updated PR notes

**Comment by pointmekin** [comment]
Updated for the Elderly and Caretaker home screens.

File: src/config/axiosConfig.ts [hunk 1]
```diff
@@ -7,7 +7,9 @@ export const client = Axios.create({
 client.interceptors.request.use(
   async (req) => {
     if (!req.headers?.Authorization) {
-      const token = (await AsyncStorage.getItem('token')) ?? '';
+      const token =
+        (await AsyncStorage.getItem('token')) ??
+        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjUxLCJyb2xlIjoiQ2FyZXRha2VyIiwiaWF0IjoxNjQ4MzcwODU2LCJleHAiOjE2NDg5NzU2NTZ9.g730zlMLsPh-bAK7OrDIlJ-8ozMRAXJ1jSWRNuL1MGk';
```
Comment (bweam) [inline src/config/axiosConfig.ts]: Please remove JWT

**Review by bweam — CHANGES_REQUESTED** [review]

File: src/config/axiosConfig.ts (hunk 1 above)
Comment (pointmekin) [reply src/config/axiosConfig.ts]: removed

**Review by bweam — APPROVED** [review]
LGTM

**Comment by ImFelippe365** [comment]
can you show us how to use the connected steps? i have the same problem, but idk use this library

**Pull request merged**